tools/rosconsole/CMakeLists.txt

```diff
@@ -121,3 +121,8 @@ if(CATKIN_ENABLE_TESTING)
     target_link_libraries(${PROJECT_NAME}-thread_test ${PROJECT_NAME})
   endif()
 endif()
+
+catkin_python_setup()
+
+catkin_install_python(PROGRAMS scripts/rosconsole
+  DESTINATION ${CATKIN_PACKAGE_BIN_DESTINATION})
```

Comment:
It probably makes sense to install this into the global bin folder rather then the package-specific libexec folder to be able to run it without `rosrun`. Instead of using `catkin_install_python` you can list it as a script in the setup.py file.